### PR TITLE
Fixed CI for optitrack

### PR DESCRIPTION
**Pull request by aarsht7**
Hello devs,

I have fixed CI for optitrack driver. Currently, it is skipping the tests because 5/8 tests are failing. Specifically, CTest is giving error and due to which overall result is failing while it is completing the building successfully.

LMK if I can help fix the CI + change CMakelist + prepare install_sdk file for vicon as well.

Cheers.

### CODE REVIEW

**Comment by aarsht7** [comment]
I was unable to setup badge on the Readme file. I am not sure what the issue can be.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # mocap_optitrack_driver
 
-[![GitHub Action
-Status](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver/workflows/main/badge.svg)](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver)
+[![GitHub Action Status](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver/workflows/main/badge.svg)](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver)
+
+[![humble Status](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver/workflows/main/badge.svg)](https://github.com/MOCAP4ROS2-Project/mocap4ros2_optitrack/tree/main)
```
Comment (fmrico) [inline README.md:5]: https://github.com/PlanSys2/ros2_planning_system/actions/workflows/humble-devel.yaml/badge.svg

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # mocap_optitrack_driver
 
-[![GitHub Action
-Status](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver/workflows/main/badge.svg)](https://github.com/MOCAP4ROS2-Project/mocap_optitrack_driver)
```
Comment (fmrico) [inline README.md:4]: https://github.com/MOCAP4ROS2-Project/mocap4ros2_optitrack/actions/workflows/main.yaml/badge.svg

File: README.md (hunk 1 above)
Comment (fmrico) [inline README.md:5]: We still haven't a `humble-devel` devel branck. We should only show `main`status

Comment (aarsht7) [reply README.md:5]: Yes, CI is currently testing on humble so I though of that. I will change to main 👍🏻

**Comment by fmrico** [comment]
Merging :rocket: 

Thanks @aarsht7

**Pull request merged**